src/pages/publication/description_tab/DisciplineSearch.tsx

```diff
@@ -5,7 +5,7 @@ import { AutoSearch } from '../../../components/AutoSearch';
 import disciplines from '../../../utils/testfiles/disciplines_en.json';
 
 interface DisciplineSearchProps {
-  setFieldValue: (name: string, value: any) => void;
+  setFieldValue: (value: any) => void;
 }
```

Comment:
er det egentlig nødvendig å få denne fra props? kan vi ikke heller bruke useFormikContext her? isteden for i parent?

Reply:
Da vil vi knytte denne komponenten opp mot formik, og ett bestemt field name i formik (kan evt sende field name som prop, da). Tenker det kan være greit å ha en generell metode som bare setter verdi, så kan parent sende inn hva som helst, om det er egen setState, redux action, eller set formik field

Reply:
navnet er kanskje litt misvisende da. Bedre med `setValueFunction`?

Reply:
da skjønner jeg. jeg tenkte at hvis det uansett er en funksjon som skal gjøre et eller annet med Formik, så kunne vi like så godt kalt setFieldValue i den nærmeste komponenten som trenger det, men hvis tanken er at man kan sende inn hva som helst, så er det good 👍 kunne kanskje endret navn ja, sånn at vi skiller på hva som er Formik-spesifikt og ikke

Reply:
yes. i do dat